back-end/src/main/java/server/BoxRequestHandler.java

```diff
@@ -138,7 +142,7 @@ private String checkParameters(Map<String, String> queryMap) {
         return "";
     }
 
-    private void sendStats(String queryType, double seconds, int fetchedRows) {
+    public static void sendStats(String queryType, double seconds, int fetchedRows) {
         String sql =
```

Comment:
minor: I feel it is cleaner if we put `sendStats` method as a public static method of `Server.java`. After all conceptually it's not something that "belongs" to `BoxRequestHandler`.